support/cas-server-support-token-tickets/src/main/java/org/apereo/cas/token/authentication/principal/TokenWebApplicationServiceResponseBuilder.java

```diff
@@ -58,6 +60,16 @@ protected WebApplicationService buildInternal(final WebApplicationService servic
 
         return jwtService;
     }
+    
+    private boolean isTicketIdAvailable(final Map<String, String> parameters){
+        final String ticketId=parameters.get(CasProtocolConstants.PARAMETER_TICKET);
+
+        if (StringUtils.isBlank(ticketId)){
+            return false;
+        }
+
+        return true;
+    }
```

Comment:
Simplify to one line:

`return StringUtils.isNotBlank(parameters.get(CasProtocolConstants.PARAMETER_TICKET))`

Reply:
Also fix formatting issues

Reply:
Done.